zp-relayer/endpoints.ts

```diff
@@ -241,6 +242,25 @@ async function getLimits(req: Request, res: Response) {
   res.json(limitsFetch)
 }
 
+function getSiblings(req: Request, res: Response) {
+  const errors = checkGetSiblings(req.query)
+  if (errors) {
+    logger.info('Request errors: %o', errors)
+    res.status(400).json({ errors })
+    return
+  }
+
+  const index = req.query.index as unknown as number
+
+  if (index >= pool.state.getNextIndex()) {
```

Comment:
Is it possible that newly registering user will observe a new tree root faster than the relayer? This case will trigger an error on the relayer and the user fast sync will not work, right?

Reply:
>Is it possible that newly registering user will observe a new tree root faster than the relayer? 

As client primarily communicates with the relayer (and not the contract directly), and they fetch the root before requesting siblings, this should not be possible. Am I missing something?

Anyways, if we try to request siblings removing this check and `index` will be `>= next_index` we will get an `libzkbob-rs` error `Tree is undefined at index ${index}`

Reply:
> they fetch the root before requesting siblings, this should not be possible

If that's the case, then we are good. Just wanted to make sure that client will not get any weird errors.